src/midi/midi_mt32.cpp

```diff
@@ -626,6 +630,15 @@ bool MidiHandler_mt32::Open([[maybe_unused]] const char *conf)
 		mixer_channel->SetLowPassFilter(FilterState::Off);
 	}
 
+	// Double the baseline PCM prebuffer because MIDI is demanding
+	const auto render_ahead_ms = MIXER_GetPreBufferMs() * 2;
+
+	static constexpr auto max_frames_per_ms = 48;
+	frame_fifo.Resize(render_ahead_ms * max_frames_per_ms);
+
+	static constexpr auto max_midi_msg_per_ms = 255;
```

Comment:
I don't think we are in a position to make such an assumption. That seems very arbitrary and we might run into problems when the actual stream of MIDI events exceeds that limit.

Reply:
Also, the revised value calls for a comment (i.e. a short version of the above explanation on choosing a reasonable upper limit given the constraints of the MIDI protocol).

Reply:
Added both; thanks for these details @johnnovak !